requirements.txt

```diff
@@ -1 +1,2 @@
+orjson
```

Comment:
To reduce the likelihood that future 'orjson' API updates break this package, can you pin the 'orjson' version?

Reply:
Might not be the best idea to pin orjson to a specific version as its widely used including by developers building webapps with fastapi / starlette. Pinning the version here might cause dependency conflicts with those apps when future version are released or if someone is still on an older version for some reason. Orjson's api has been stable for a long time and is unlikely to change unless the json standard library changes. Maybe we could do something like `orjson>=3.6.0,<4.0.0`?